lib/PGalias.pm

```diff
@@ -205,41 +206,36 @@ sub make_alias {
 	# dispatch	
 #######################################################################
 	# determine extension, if there is one
-	# if extension exists, strip and use the value for $ext
+	# if extension exists use the value for $ext
 	# files without extensions are flagged with errors.
-
-	#      								$self->debug_message("This auxiliary file id is $aux_file_id" );
-	if ($aux_file_id =~ s/\.([^\.]+)$// ) {
+	# The extension is retained as part of  aux_file_id 
+	
+	#$self->debug_message("This auxiliary file id is $aux_file_id" );
+	if ($aux_file_id =~ m/\.([^\.]+)$/ ) {
 		$ext = $1;
 	} else {
 		$self->warning_message( "The file name $aux_file_id did not have an extension.<BR> " .
 		     "Every file name used as an argument to alias must have an extension.<BR> " .
-		     "The permissable extensions are .jpg, .pdf, .gif, .png, and .html .<BR>");
+		     "The permissable extensions are .jpg, .pdf, .gif, .png, .mpg, .mp4, .ogg, .webm and .html .<BR>");
 		$ext  = undef;
```

Comment:
My changes from pull #264 seem to have snuck in here some how.  If this gets pulled before #264 then #264 will be moot.